android-configuration.gradle

```diff
@@ -41,6 +41,10 @@ android {
     warningsAsErrors true
     disable 'UnusedResources' // we're exposing resources for consumers
   }
+
+  buildFeatures {
```

Comment:
This stops generating a `buildConfig` file, which was also generated in java and we don't need it.